jsapp/scss/components/_kobo.form-builder.scss

```diff
@@ -262,21 +262,17 @@ $t-form-builder-aside-open: 200ms;
   }
 }
 
-.formBuilder-header__button {
+.form-builder-header__button {
   @extend .mdl-button;
 
-  > i {
-    @extend .fa, .fa-fw;
-  }
-
   &:focus {
     i {
       @include form-builder-focus();
     }
   }
 
-  &.formBuilder-header__button--create,
-  &.formBuilder-header__button--save {
+  &.form-builder-header__button--create,
+  &.form-builder-header__button--save {
```

Comment:
Do you think we should try to move loading integrated buttons to something more general? I think it would be useful in a lot of places that would need a `confirm` button that makes some sort of request

Reply:
We could try making `kobo-button--loading` and `kobo-light-button--loading` modifiers. It could either make a spinner appear as `::before` and some space for it inside button (this has some tricky parts, like hiding existing icon and handling some edge cases) or maybe some other way to animate the button indicating an operation in progress. 

Currentl I think there few different approaches: button text changing, spinner icon appearing, disabling.

Having a single way to do it would be useful for sure.

Reply:
Made an issue for it https://github.com/kobotoolbox/kpi/issues/3133